backend/Hortogram/Hortogram/Business/UserService.cs

```diff
@@ -93,5 +129,68 @@ public bool UpdateUser(Guid id, string firstName, string lastName, string email,
                 return false;
             }
         }
+
+        public List<UserFollowersResponse> GetFollowers(Guid userId)
+        {
+            try
+            {
+                User user = Utils.ConvertUserResponseToUser(GetById(userId));
+
+                if (user == null)
+                    return null;
+
+                List<UserFollowersResponse> followers = UserRepository.GetFollowers(user);
+                return followers;
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                return null;
+            }
+        }
+
+        public bool AddFollower(Guid userId, Guid followerId)
+        {
+            try
+            {
+                if (userId == followerId)
+                    return false;
+
+                User user = UserRepository.GetById(userId);
+                User follower = UserRepository.GetById(followerId);
+
+                if (user == null || follower == null)
+                    return false;
+
+                UsersFollowers userFollower = new UsersFollowers { UserId = userId, FollowerId = followerId };
+                UserRepository.AddFollower(userFollower);
+                return true;
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                return false;
+            }
+        }
+
+        public bool RemoveFollower(Guid userId, Guid followerId)
+        {
+            try
+            {
+                User user = UserRepository.GetById(userId);
+                User follower = UserRepository.GetById(userId);
```

Comment:
Acredito que você trocou o id aqui, seria:

```suggestion
                User follower = UserRepository.GetById(followerId);
```

correto?

Reply:
Isso mesmo mano, acabei trocando